okhttp/src/main/java/okhttp3/internal/http2/Hpack.java

```diff
@@ -32,7 +31,7 @@
 /**
  * Read and write HPACK v10.
```

Comment:
I assume the v10 was a draft

```suggestion
 * Read and write HPACK.
```